<launch>

  	<include file="$(find suturo_manipulation_moveit)/launch/tf.launch"/>

  	<node name="arm_controller" pkg="suturo_manipulation_moveit" type="arm_controller_servo" respawn="true" output="screen"/>

  	<node name="gripper_controller" pkg="suturo_manipulation_moveit" type="gripper_controller" respawn="true" output="screen"/>

	<include file="$(find suturo_manipulation_moveit)/launch/move_group.launch">
  		<arg name="publish_monitored_planning_scene" value="true" />
 	</include>

	<include file="$(find suturo_manipulation_moveit)/launch/moveit_rviz.launch"/>

</launch>